src/main/resources/org/jenkinsci/plugins/workflow/libs/Messages.properties

```
@@ -24,3 +24,9 @@ LibraryDecorator.could_not_find_any_definition_of_librari=Could not find any def
 ResourceStep.library_resource_ambiguous_among_librari=Library resource {0} ambiguous among libraries {1}
 ResourceStep.no_such_library_resource_could_be_found_=No such library resource {0} could be found.
 SCMSourceRetriever.library_path_no_double_dot=Library path may not contain ".."
+GlobalLibraries.DisplayName=Global Trusted Pipeline Libraries
+GlobalLibraries.Description=Sharable libraries available to any Pipeline jobs running on this system. \
+    These libraries will be trusted, meaning they run without \u201csandbox\u201d restrictions and may use <code>@Grab</code>.
+GlobalUntrustedLibraries.Description=Sharable libraries available to any Pipeline jobs running on this system. \
+    These libraries will be untrusted, meaning they run with \u201csandbox\u201d restrictions and can''t use <code>@Grab</code>.
```

```suggestion
GlobalLibraries.Description=Sharable libraries available to any Pipeline jobs running on this system. \
    These libraries will be trusted, meaning they run without \u201csandbox\u201d restrictions and may use <code>@Grab</code>.
GlobalUntrustedLibraries.Description=Sharable libraries available to any Pipeline jobs running on this system. \
    These libraries will be untrusted, meaning they run with \u201csandbox\u201d restrictions and cannot use <code>@Grab</code>.
```
and BTW you can use UTF-8 now